tests/test_admin_endpoints_97.py

```
@@ -18,7 +18,21 @@
 from starlette.types import ASGIApp
 from typing import cast
 
-from app import app
+import sys
+import os
```

Ruff (0.13.1)

Redefinition of unused `os` from line 13

Remove definition: `os`

(F811)